src/opm/parser/eclipse/share/keywords/keyword_list.cmake

```diff
@@ -9,6 +9,25 @@
 # you are therefore encouraged to go to the opm-simulators repository and update
 # the file: opm/autodiff/MissingFeatures.cpp
 
+#Some keywords are found to be of 'special' structure:
+#W             WHAT
+#ECONT         Terminates with two /
+#COALNUM       matrix structure
+#COMPDATM      alias for COMPDATL
```

Comment:
You should only add one json file - with an *arbitrary name*, and then include the `deck_names` attribute:
```json
"deck_names":  ["COMPDATM","COMPDATL"], 
```
 see e.g. the json file [`W/WELL_PROBE`](https://github.com/OPM/opm-common/blob/master/src/opm/parser/eclipse/share/keywords/000_Eclipse100/W/WELL_PROBE)